internal/pkg/mount/pseudo.go

```diff
@@ -35,7 +35,9 @@ func PseudoSubMountPoints() (mountpoints *Points, err error) {
 
 	if _, err := os.Stat(constants.EFIVarsMountPoint); err == nil {
```

Comment:
i think the rockpi board lies about efivars being present to the kernel :sweat_smile:

Reply:
yeah, that might explain why it works on other boards :)